scilpy/io/utils.py

```diff
@@ -22,6 +22,16 @@ def check_tracts_same_format(parser, filename_list):
             parser.error('All tracts file must use the same format.')
 
 
+def add_json_arg(parser):
+    g1 = parser.add_argument_group(title='Json options')
+    g1.add_argument('--indent',
+                    type=int, default=2,
+                    help='Indent for json pretty print.')
+    g1.add_argument('--sort_keys',
```

Comment:
Never used. I guess it's for the future in tractometry?

Reply:
exactly